api/r/cellxgene.census/tests/testthat/test-open.R

```diff
@@ -1,5 +1,5 @@
 test_that("open_soma", {
-  coll <- open_soma("latest")
+  coll <- open_soma("stable")
```

Comment:
I think outside of the specific `open_soma_stable` test, we should rely on latest, since that would catch potential issues with the built version earlier. It's also how we do it in Python. @atolopko-czi thoughts?